src/software/simulation/standalone_simulator.h

```diff
@@ -125,6 +128,13 @@ class StandaloneSimulator
     // simulation looking appropriately / usefully slow
     static constexpr double DEFAULT_SLOW_MOTION_MULTIPLIER = 14.0;
 
+    /**
+     * Returns the robot constants for this robot state
+     *
+     * @return the robot constants for this robot
```

Comment:
What is `this robot`? This is a bit confusing since `this` usually refers to the instance this member function is a called on (so in this case, an instance of `StandaloneSimulator`, which is very much not a robot).

Reply:
fixed